src/main/java/appeng/parts/p2p/PartP2PTunnelNormal.java

```diff
@@ -73,10 +72,12 @@ public boolean onPartActivate(final EntityPlayer player, final Vec3 pos) {
                 }
             }
             mc.notifyUser(player, MemoryCardMessages.INVALID_MACHINE);
-        } else if (!player.isSneaking() && is != null && is.getItem() instanceof IToolWrench && !Platform.isClient()) {
-            printConnectionInfo(player);
-        } else if (tt != null) // attunement
-        {
+        } else if (!player.isSneaking()
+                && Platform.isWrench(player, is, (int) pos.xCoord, (int) pos.yCoord, (int) pos.zCoord)
+                && !Platform.isClient()) {
+                    printConnectionInfo(player);
+                    // spotless:off
```

Comment:
nit: alignment

Reply:
spotless formats `// spotless:off` to here...

Reply:
ive been pretty upset with spotless lately